homeassistant/components/risco/config_flow.py

```diff
@@ -51,6 +52,7 @@
         vol.Required(CONF_HOST): str,
         vol.Required(CONF_PORT, default=1000): int,
         vol.Required(CONF_PIN): str,
+        vol.Optional(CONF_COMMUNICATION_DELAY, default=0): int,
```

Comment:
This belongs in an options flow, not a configuration flow.

Reply:
This has been added to the options now and I also made some changes according to the discussion above.
Since OnFreund won't be able to do a code review in the near future, do we keep this PR as draft until he is available again or can we proceed without his approval?